systemd/osfingermap.yaml

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+#
+# Setup variables using grains['osfinger'] based logic.
+# You just need to add the key:values for an `osfinger` that differ
+# from `defaults.yaml` + `os_family.yaml` + `osmap.yaml`.
+# Only add an `osfinger` which is/will be supported by the formula
+# (empty `osfinger`s do not need to be listed, just added here as an example).
+---
+Ubuntu-18.04: {}
+CentOS-7: {}
```

Comment:
Some formulas use `osfinger` mappings and others use `oscodename` mappings.  I can see advantages to both.  The latter is particularly useful when pulling packages from repos, such as in the [`postgres-formula`](https://github.com/saltstack-formulas/postgres-formula/blob/master/postgres/codenamemap.yaml).  Can't think of a way of using both.  What should be the base recommendation for formulas?

Reply:
Perhaps both types should be left out of formulas until they are actually needed?  Then make the selection at the time of need?  This is a question more than a suggestion.

Reply:
I just added them for reference, so if one wants to contribute they can see where it could go.

Reply:
@aboe76 Yes, that's fine.  But long-term, which is more advantageous -- `osfinger` or `oscodename`?  Can't use both together, right?